script/gen_requirements_all.py

```diff
@@ -257,6 +268,22 @@ def gather_modules():
     return reqs
 
 
+def process_requirements(errors, module_requirements, package, reqs):
+    """Process all of the requirements."""
+    for req in module_requirements:
+        if req in IGNORE_REQ:
+            continue
+        if '://' in req and 'pyharmony' not in req:
```

Comment:
no more need to exclude `pyharmony` 

it has been replaced by pypi lib `aioharmony`